react/example-customer-account--loyalty-app--react/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "loyalty-app-react",
+  "version": "1.0.0",
+  "description": "Customer account loyalty app example",
+  "main": "index.js",
+  "scripts": {
+    "shopify": "shopify",
+    "build": "shopify app build",
+    "dev": "shopify app dev",
+    "info": "shopify app info",
+    "generate": "shopify app generate"
+  },
+  "author": "Shopify",
+  "license": "MIT",
+  "dependencies": {
+    "@shopify/app": "^3.50.0",
+    "@shopify/cli": "^3.50.0",
+    "@shopify/ui-extensions": "0.0.0-unstable-20231024205236",
+    "@shopify/ui-extensions-react": "0.0.0-unstable-20231024205236",
```

Comment:
My concern is that developers might use this code as a starting point for their extensions (it seems many have done this for Checkout extensions), and if we hardcode this to a specific release of `unstable`, these developers would not be getting our changes without manually updating this.

Reply:
I understand that although there is also a risk that with updates to unstable, the code will not work at all out the box if there are breaking changes which can be frustrating because now they will have to update the code themselves. 

I do think having to update the package.json is better than the risk of having broken / not running code 🤔 perhaps?

Reply:
It could also create more maintenance for us because now we have to track each release of unstable and make sure the tutorials still works. I dont think this ideal place to be in?

Reply:
I guess if we are okay with consistently tophatting the tutorials when there is a PR that will merge to unstable for customer accounts, we can do this?

Reply:
I think we should but this also means any team that makes updates to unstable the tutorials will get latests changes. This does not only apply to us. I think this is a bit risky?